specifyweb/frontend/js_src/lib/components/WorkBench/Status.tsx

```diff
@@ -134,6 +134,44 @@ export function WbStatus({
       </>
     );
 
+  const startTime = React.useRef(Date.now());
+  const [remainingTime, setRemainingTime] = React.useState<number | null>(null);
+
+  // Convert seconds to minutes and seconds and create returned string
```

Comment:
This function should not be defined inside a component as it doesn't have any logic specific to this component - it's just a utility. Move it into `./Atoms/Internalization.tsx` or `./utils/utils.ts`